queue_services/entity-filer/tests/unit/filing_processors/test_dissolution.py

```diff
@@ -92,14 +93,33 @@ def test_dissolution(app, session, minio_server, legal_type, identifier, dissolu
     party_role.save()
     curr_roles = len(business.party_roles.all())
 
+    if dissolution_type == 'involuntary':
+        batch = Batch(
+            batch_type=Batch.BatchType.INVOLUNTARY_DISSOLUTION,
+            status=Batch.BatchStatus.PROCESSING,
+            size=1,
+        )
+        batch.save()
+        batch_processing = BatchProcessing(
+            batch_id=batch.id,
+            business_id=business.id,
+            business_identifier=business.identifier,
+            step=BatchProcessing.BatchProcessingStep.DISSOLUTION,
+            status=BatchProcessing.BatchProcessingStatus.QUEUED,
+            created_date=datetime.utcnow()-datedelta(days=42),
+            trigger_date=datetime.utcnow(),
+            last_modified=datetime.utcnow()
+        )
+        batch_processing.save()
```

Comment:
you said your test works?  don't you need the filing id to be populated here?

Reply:
Just reran the tests for this file and yeah they still do, but good point I feel like that does need to be populated. Having a look into what's going on now.

<img width="213" alt="Screenshot 2024-08-14 at 1 50 01 PM" src="https://github.com/user-attachments/assets/15a48bc6-f800-403f-b408-ff751d5bf827">

Reply:
Figured out what's going on. If `BatchProcessing.find_by` is called with `None` for any of the parameters, then it will return a list of **all** batch_processings. 

<img width="579" alt="Screenshot 2024-08-14 at 2 02 30 PM" src="https://github.com/user-attachments/assets/7a66e80e-d686-4d60-8e5e-de0941419f69">

Since only one batch_processing exists for a given test case, the tests were still passing properly. Anyway, I've updated the unit tests to set up the batch_processing correctly.

Reply:
that makes sense but i still expected the test to fail in that scenario.  i think the issue might be how we are grabbing the dissolution filing id.

Reply:
Yeah that could be it, I updated to grab it from `filing_rec` based on your comment below